recipes/doxygen/all/conanfile.py

```diff
@@ -11,6 +11,14 @@ class DoxygenConan(ConanFile):
     license = "GPL-2.0-or-later"
     url = "https://github.com/conan-io/conan-center-index"
     settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "build_parse": [True, False],
+        "build_search": [True, False]
```

Comment:
A `build_` prefix only makes sense when building, but is unneeded once available.
```suggestion
        "enable_parse": [True, False],
        "enable_search": [True, False]
```